examples/text_analysis_mosaicml_mlflow.py

```diff
@@ -99,35 +99,36 @@
 
 # COMMAND ----------
 
-# get or create mosaic route
-import mlflow
-from mlflow import gateway
+# # get or create mosaic route
```

Comment:
You can replace this cell with a `%md` cell pointing to the route setup notebook

Reply:
And please put the same instructions in the other notebook